src/python/WMCore/Services/WMStats/WMStatsReader.py

```diff
@@ -248,19 +248,19 @@ def _getAgentInfo(self):
         
         return result
     
-    def agentsByTeam(self, ignoreDrain = True):
+    def agentsByTeam(self, filterDrain=True):
```

Comment:
Actually, in initial usage, we filtered out drain agent for calculating the resource distribution in AgentWatcher component. I can change those if it makes more sense including drain agent for team.